app/src/main/res/layout/topic_fragment.xml

```diff
@@ -41,13 +41,25 @@
         app:tabTextColor="@color/tab_icon_color_selector" />
     </com.google.android.material.appbar.AppBarLayout>
 
-    <androidx.viewpager.widget.ViewPager
-      android:id="@+id/topic_tabs_viewpager"
+    <FrameLayout
+      android:id="@+id/topic_tabs_viewpager_container"
       android:layout_width="match_parent"
       android:layout_height="0dp"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@+id/topic_app_bar_layout" />
+      app:layout_constraintTop_toBottomOf="@+id/topic_app_bar_layout">
+
+      <androidx.viewpager.widget.ViewPager
+        android:id="@+id/topic_tabs_viewpager"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+      <View
```

Comment:
Nit: Rename the widgets as per the file name or please remove it if its not being used

Reply:
I have filed an issue for this because we need to fix this in other files too.
https://github.com/oppia/oppia-android/issues/556